server/internal/nbtconv/mapread.go

```diff
@@ -78,11 +78,12 @@ func MapBlock(x map[string]any, k string) world.Block {
 // to a world.Item.
 func MapItem(x map[string]any, k string) item.Stack {
 	if m, ok := x[k].(map[string]any); ok {
-		s := readItemStack(m)
-		readDamage(m, &s, true)
-		readEnchantments(m, &s)
-		readDisplay(m, &s)
-		readDragonflyData(m, &s)
+		t := Map[map[string]any](m, "tag")
```

Comment:
Is this correct for every item in Bedrock Edition? I feel it's weird we didn't have this before, so we need to make sure this matches Bedrock Edition.